services/base-jdbc/src/main/resources/org/eclipse/hono/service/base/jdbc/store/tenant/base.sql.yaml

```diff
@@ -99,3 +99,11 @@ updateVersioned: |
    AND
       version=:expected_version
 
+getTenants: |
```

Comment:
Maybe we can already call this *searchTenants* or *findTenants* in anticipation of the search params being supported in the near future?

Reply:
done